doc/releases/release-notes-3.7.rst

```diff
@@ -162,6 +162,23 @@ Architectures
 
 * Xtensa
 
+  * Added support to save/restore HiFi AudioEngine registers.
+
+  * Added support to utilize MPU.
+
+  * Added support to automatically generate interrupt handlers.
+
+  * Added support to generate vector table at build time to be included in the linker script.
+
+  * Added kconfig :kconfig:option:`CONFIG_XTENSA_BREAK_ON_UNRECOVERABLE_EXCEPTIONS` to guard
+    using break instruction for unrecoverable exceptions. The break instruction may result
```

Comment:
```suggestion
    using break instruction for unrecoverable exceptions. Without this, the break instruction may result
```
maybe better?

Reply:
Actually the kconfig enables using of break instruction. So I changed the wording a bit.